src/webui/www/private/scripts/download.js

```diff
@@ -68,7 +68,7 @@ window.qBittorrent.Download = (function() {
         $("startTorrent").checked = !pref.add_stopped_enabled;
         $("addToTopOfQueue").checked = pref.add_to_top_of_queue;
 
-        if (pref.auto_tmm_enabled === 1) {
+        if (pref.auto_tmm_enabled === true) {
```

Comment:
How about:
```suggestion
        if (pref.auto_tmm_enabled) {
```

Reply:
It reverts the check back to a loose comparison. I don't mind though, if you are ok with it.

Reply:
>It reverts the check back to a loose comparison. I don't mind though, if you are ok with it.

I just checked the origin of it, seems `auto_tmm_enabled` is really intended to be a boolean. So yes.

Reply:
Ok.